src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -435,9 +436,21 @@ internal NamespaceOrTypeOrAliasSymbolWithAnnotations BindNamespaceOrTypeOrAliasS
                     return bindPointer();
 
                 case SyntaxKind.FunctionPointerType:
+                    var functionPointerTypeSyntax = (FunctionPointerTypeSyntax)syntax;
+                    if (GetUnsafeDiagnosticInfo(sizeOfTypeOpt: null) is CSDiagnosticInfo info)
+                    {
+                        var @delegate = functionPointerTypeSyntax.DelegateKeyword;
+                        var asterisk = functionPointerTypeSyntax.AsteriskToken;
+                        var length = @delegate.Width + @delegate.TrailingWidth +
+                                     asterisk.LeadingWidth + asterisk.Width;
+
+                        RoslynDebug.Assert(@delegate.SyntaxTree is object);
+                        diagnostics.Add(info, Location.Create(@delegate.SyntaxTree, new TextSpan(@delegate.SpanStart, length)));
```

Comment:
>new TextSpan(@delegate.SpanStart, length [](start = 84, length = 40)

```FromBounds(@delegate.SpanStart, asterisk.SpanEnd)```?

Reply:
Thanks, I looked for a helper and apparently missed it.

---
In reply to: [376166854](https://github.com/dotnet/roslyn/pull/41455#discussion_r376166854) [](ancestors = 376166854)